service/src/chain_spec.rs

```diff
@@ -191,8 +190,8 @@ pub fn testnet_genesis(
 		]
 	});
 
-	const ENDOWMENT: u128 = 10_000_000 * DOLLARS;
-	const STASH: u128 = 100 * DOLLARS;
+	const ENDOWMENT: u128 = 1_000_000 * DOTS;
```

Comment:
@jacogr this must be enough funds to prevent the UI issues hopefully?

Reply:
It looks good, so should be funded enough to send txs.